universal-login-relayer/src/core/services/validators/EmailConfirmationValidator.ts

```diff
@@ -1,15 +1,23 @@
 import moment from 'moment';
 import {ensure, EmailConfirmation} from '@unilogin/commons';
-import {DuplicatedEmailConfirmation, InvalidCode, CodeExpired} from '../../utils/errors';
+import {DuplicatedEmailConfirmation, InvalidCode, CodeExpired, EmailNotConfirmed} from '../../utils/errors';
 
 export class EmailConfirmationValidator {
   constructor(
     private codeDurationInMinutes: number = 5,
   ) {}
 
+  validateEmail(emailConfirmation: EmailConfirmation) {
```

Comment:
`isConfirmed` or `validateIsConfirmed`. We don't validate email here but if it is confirmed

Reply:
Done